lms/templates/instructor/instructor_dashboard_2/certificate-invalidation.underscore

```diff
@@ -3,9 +3,16 @@
 </p>
 
 <div class="add-certificate-invalidation">
-    <input class='student-username-or-email' id="certificate-invalidation-user" type="text" placeholder="<%- gettext('Username or email address') %>">
-    <textarea class='notes-field' id="certificate-invalidation-notes" rows="10" placeholder="<%- gettext('Add notes about this learner') %>"></textarea>
+    <div>
+        <label for="certificate-invalidation-user"><%- gettext('Username or email address') %></label>
```

Comment:
For instance, here could you nest the input within the label so that the ID is not necessary? I suppose though this is not a new ID....